L1Trigger/L1THGCal/python/hgcalBackEndLayer1Producer_cfi.py

```diff
@@ -3,18 +3,64 @@
 
 import SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi as digiparam
 import RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi as recoparam
-import RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi as recocalibparam 
+import RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi as recocalibparam
 from . import hgcalLayersCalibrationCoefficients_cfi as layercalibparam
 
-C2d_parValues = cms.PSet( clusterType = cms.string('dummyC2d'),
-                          calibSF_cluster=cms.double(1.),
-                          layerWeights = layercalibparam.TrgLayer_weights,
-                          applyLayerCalibration = cms.bool(True)
-            )
+from Configuration.Eras.Modifier_phase2_hgcalV9_cff import phase2_hgcalV9
 
-be_proc = cms.PSet( ProcessorName  = cms.string('HGCalBackendLayer1Processor2DClustering'),
-                    C2d_parameters = C2d_parValues.clone()
-                  )
+c2d_calib_pset = cms.PSet(calibSF_cluster=cms.double(1.),
+                          layerWeights=layercalibparam.TrgLayer_weights,
+                          applyLayerCalibration=cms.bool(True))
+
+c2d_thresholds_pset = cms.PSet(seeding_threshold_silicon=cms.double(5.),
+                               seeding_threshold_scintillator=cms.double(5.),
+                               clustering_threshold_silicon=cms.double(2.),
+                               clustering_threshold_scintillator=cms.double(2.))
+
+# V9 samples have a different defintiion of the dEdx calibrations. To account for it
+# we reascale the thresholds for the clustering
+# (see https://indico.cern.ch/event/806845/contributions/3359859/attachments/1815187/2966402/19-03-20_EGPerf_HGCBE.pdf
+# for more details)
+phase2_hgcalV9.toModify(c2d_thresholds_pset,
+                        seeding_threshold_silicon=cms.double(3.75),
```

Comment:
better to avoid using `cms.double()` in `toModify()`, e.g. `seeding_threshold_silicon = 3.75` (to check automatically if the assigned value has the right type vs. the original value)